handlers/svg-open-street-map/osm-svg.py

```diff
@@ -35,7 +35,7 @@ def handle():
         request_schema = json.load(f)
     with open("./schemas/handler-response.schema.json") as f:
         response_schema = json.load(f)
-    with open("./schemas/renderers/osm_svglayers.schema.json") as f:
+    with open("./schemas/renderers/svglayers.osm.schema.json") as f:
```

Comment:
Question - why is there a new schema here?

Reply:
@JRegimbal, You would recall that there was an error when the visual debug handler for OSM was tested in the morning today.  I checked through to realize that it was due to an inconsistency in the schema file name, so the path could not be found. Here I used "osm_svglayers.schema.json" while the actual file name in the schemas’ directory was "svglayers.osm.schema.json". So, all I needed to do was to change either of the names to be the same.